ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -76,18 +80,33 @@ class AlertTabs extends Component {
           this.refreshKapacitorConfig(this.props.kapacitor)
           this.props.addFlashMessage({
             type: 'success',
-            text: `Alert for ${section} successfully saved`,
+            text: `Alert configuration for ${section} successfully saved.`,
           })
         })
         .catch(() => {
           this.props.addFlashMessage({
             type: 'error',
-            text: 'There was an error saving the kapacitor config',
+            text: `There was an error saving the alert configuration for ${section}.`,
           })
         })
     }
   }
 
+  handleTestConfig = section => async () => {
+    try {
+      await testAlertOutput(this.props.kapacitor, section, {})
```

Comment:
Looks like `testAlertOutput` only takes two arguments now :).  Let's remove the `{}`